mix.exs

```diff
@@ -36,10 +36,10 @@ defmodule ExAws.Mixfile do
       {:xml_builder, "~> 0.0.6", optional: true},
       {:sweet_xml, "~> 0.6", optional: true},
       {:ex_doc, "~> 0.14", only: :dev},
-      {:hackney, "1.6.3 or 1.6.5 or 1.7.1 or 1.8.6", optional: true},
-      {:poison, ">= 1.2.0", optional: true},
+      {:hackney, "1.8.6", optional: true},
```

Comment:
I don't understand the point of this. I already permit 1.8.6. I'm not going to force people to upgrade just to fix dialyzer warnings.